### ISSUE TITLE
video browse loading

### ISSUE THREAD

**Issue by lordofnos** [moxi9/phpfox#1177]
when there are more than 12 videos the older ones do not load when scrolling down. tested on own site with 4.2.2 and v4.phpfox.com with ff 44.0.2

**Comment by Data66PF** [comment]
Using FF and Chrome, I see 31 videos as they all load for me at the demo.

**Comment by lordofnos** [comment]
as you said in forum i have made the test at v4.phpfox.com. at the moment i can not see any video...

**Comment by lordofnos** [comment]
on stage.phpfox.com it works for me too.

**Comment by lordofnos** [comment]
ah ok, i have to login before i see them. with my account on v4.phpfox.com i can only see the first 12 videos.

**Comment by Data66PF** [comment]
Are you sure you have friends with videos loaded? The site is set to the default friends only community.

**Comment by lordofnos** [comment]
i can see the other videos but after posting 12 more the old ones are not loaded anymore

**Comment by ashdelacroix** [comment]
I can confirm that I also have this exact problem. I'm using a fresh, clean install of the latest version. I've also tested this on multiple devices and browsers, firewalls off, ad-blockers off and so forth, at home and at my office, and nothing seems to resolve the problem.

**Comment by Data66PF** [comment]
We have v4.3 on the stage now (as of a few days ago) so you may want to test this after we release 4.3.

**Comment by ashdelacroix** [comment]
Thanks @Data66PF, that is probably the best way forward. I've been trying to think of a reason why this would happen and I am lost. Video is important to my community, so it's important I find a solution. I will await the release of 4.3.

**Comment by ashdelacroix** [comment]
Some things I've noticed in the last few hours since uploading several more videos. It doesn't look like a problem with autoscrolling in itself, but the way the app decides what or how many clips to display at once. The page had been displaying 12 videos, refusing to load more. Then I uploaded more videos, and autoscrolling then worked and loaded a further 3 thumbnails, making 15 on display at once. It seems to be showing everything it believes is "recent" but no older clips. There are several other, older videos that don't appear unless I specifically search for them or choose "most watched". Thumbnails of older videos just don't want to load on the browse page using autoscroll (but searching proves that they are in the system).

**Comment by ashdelacroix** [comment]
After uploading more and more videos, there is a definite problem here with what the browse page decides to display. Video thumbnails that were not previously displayed (as i described in my last comment) are now displaying, and some newer clips are no longer displaying. The autoscroll appears to function more or less, but the logic that decides which video thumbnails to present seems entirely inconsistent to a point of randomness. The more videos you upload, the more the randomness exists.

**Comment by Data66PF** [comment]
https://stage.phpfox.com/v/ I just added 12 videos. Can you please see if I tested it right? I have the sort set to latest for my account. You might need to have friended me to see them. That is 4.3

**Comment by Data66PF** [comment]
Actually, I shared 13 and now can see the bug. One is missing but it shows in the feed.

Staff - share over 12 videos in the video section (I tested there and not via status share). I tested using dailymotion and youtube. 

Scroll down and you'll note that some of the videos you shared are missing. I see only 12 of the 13 I just shared. On my profile, I have 16 videos but in the video section it only shows 15 in the main view.
